README.md

```diff
@@ -240,25 +240,22 @@ csi-secrets-store-provider-azure-sxht2             2/2     Running   0
 
 ## Providers
 
-This project features a pluggable provider interface developers can implement that defines the actions of the Secrets Store CSI driver.
+This project features a pluggable provider interface developers can implement that defines the actions of the Secrets Store CSI driver. This enables retrieval of sensitive objects storied an enterprise-grade external secrets store into Kubernetes while continue to manage these objects outside of Kubernetes.
 
-This enables on-demand retrieval of sensitive objects storied an enterprise-grade external secrets store into Kubernetes while continue to manage these objects outside of Kubernetes.
+### Criteria for Supported Providers
 
-Each provider may have its own required properties.
+Here is a list of criteria for being added as a supported provider:
+1. Code audit of the provider implementation to ensure it adheres to the required provider-driver interface, which includes:
+    - implementation of provider command args https://github.com/deislabs/secrets-store-csi-driver/blob/master/pkg/secrets-store/nodeserver.go#L223-L236
+    - provider binary naming convention and semver convention
+    - provider binary deployment volume path
+    - provider logs are written to stdout and stderr so they can be part of the driver logs
+1. Add provider to the e2e test suite to demonstrate it functions as expected https://github.com/deislabs/secrets-store-csi-driver/tree/master/test/bats
```

Comment:
Is the "how" for this documented in the tests, or somewhere else?

Reply:
Will update to `Add provider to the e2e test suite to demonstrate it functions as expected https://github.com/deislabs/secrets-store-csi-driver/tree/master/test/bats Please use existing providers e2e tests as a reference.`